modules/statistics/php/stats_demographic.class.inc

```diff
@@ -95,10 +98,10 @@ class Stats_Demographic extends \NDB_Form
         $tpl_data['Subcategories'] = $subcats;
         $tpl_var = \Utility::getSubprojectsForProject($projectID);
         $tpl_data['Subprojects']      = $tpl_var;
-        $tpl_data['DropdownName']     = $dropdown_name;
-        $tpl_data['DropdownOptions']  = $dropdown_opt;
-        $tpl_data['DropdownSelected'] = $dropdown_selected;
-        $tpl_data['Centers']          = $centres;
+        $tpl_data['DropdownName']     = $dropdownName;
+        $tpl_data['DropdownOptions']  = $dropdownOpt;
+        $tpl_data['DropdownSelected'] = $dropdownSelected;
+        $tpl_data['Centers']          = $centers ?? array();
```

Comment:
why does this need to be an empty array if not set?

Reply:
See `templates//form_stats_behavioural.tpl`. I think PHP will give a warning or notice if it tries to iterate over an unset or null value, whereas an empty array should be fine.